tldraw/packages/core/src/lib/TLApp/TLApp.ts

```diff
@@ -889,7 +889,7 @@ export class TLApp<
       !this.isInAny('select.translating', 'select.pinching') &&
       this.selectedShapes.size > 0 &&
       !this.selectedShapesArray.every(shape => shape.hideSelectionDetail) &&
-      false // FIXME: should we shoult the selection detail?
+      false // FIXME: should we show the selection detail?
```

Comment:
@pengx17 Is `show` the right word here?

Reply:
This is correct. Peng isn't active with the project as much